website/en/documents/connectors/README.md

```diff
@@ -32,4 +32,5 @@ dir:
 - [StreamingFile(HDFS streaming) connector](streamingfile/streamingfile.md)
 - [Fake connector](fake/fake.md)
 - [Print connector](print/print.md)
+- [Print-v1 connector](print/v1/print-v1.md)
```

Comment:
can we just replace the old one by the new one? Do you think we can deprecate the old one?